.eslintrc

```diff
@@ -0,0 +1,18 @@
+{
+  "parser": "babel-eslint",
+  "extends": "airbnb",
```

Comment:
Let's extend `@folio/eslint-config-stripes` instead.  This captures the necessary rules already.

Reply:
Make sense.